server/controllers/DataAccessLayer.js

```diff
@@ -65,7 +66,8 @@ exports.ReadUsersFile = function() {
  */
 exports.AddUserToFile = function(user) {
     // Create a string to store in the text file as a user
-    let userString = user.id + "," + user.username + "," + user.password + "," + 
+    let userString = user.id + "," + (user.isAdmin ? "true" : "false") + "," +
```

Comment:
Is `(user.isAdmin ? "true" : "false")` check needed if you're passing a boolean from `CreateNewUser`?